src/api/app/assets/stylesheets/request_show_redesign/build_results.scss

```diff
@@ -0,0 +1,23 @@
+.results {
+  .result {
+    width: calc(100% - 1rem);
+    border-left: thick solid $gray-300;
+    cursor: help;
+
+    &:hover {
+      background-color: $gray-100;
+    }
+
+    @include media-breakpoint-up(sm) {
+      width: calc(50% - 3.5rem);
```

Comment:
Where is `3.5.rem` coming from?

Reply:
The addition of the left and right margins applied to the result box.

Reply:
A comment or even better, some Sass variables would make this obvious. Otherwise, it looks like a magic number and in a few months from now, we don't know anymore what this number refers to.

Reply:
I added some comments